### ISSUE TITLE
Here's how to do hand gesture recognition:

### ISSUE THREAD

**Issue by StealthySemicolon** [carykh/jumpcutter#44]
Make a deep learning model to detect hand
Take pictures of thumbs up and thumbs down hands
Make sure they are square
Train new deep learning model on this data
???
Profit

**Comment by DeflatedPickle** [comment]
The project's gotten 365 stars so far, I'm sure we can all put our hands together and sort something out!

**Comment by Puppy288** [comment]
what about foot gestures

**Comment by DoomRater** [comment]
How about coloring thumbs up hands green and coloring thumbs down pictures red, and training to detect these symbols instead?  We could standardize the image and have people print out cards using the same images that the model was trained on.

**Comment by StealthySemicolon** [comment]
@DoomRater YES! Amazing idea. We can use a colour filter on OpenCV to see if there are more red pixels or green pixels and use that to determine if we want to go or no go.

**Comment by Tymski** [comment]
Or make a QR code on a stick with one side being OK and the other side being NEY, should be pretty easy with so many QR code reading libraries

**Comment by DataStrategist** [comment]
@DoomRater yes, I suggested the same in youtube... if we all use the same image it would simplify the problem.

Otherwise, could we crowdsource a thumbsup/thumbsdown corpus for him? :D

**Comment by DeflatedPickle** [comment]
@DataStrategist, crowdsourced thumbs up/down to throw into a model seems like the best route.
